client/tests/webdriver/specs/pendingAssessments.spec.js

```diff
@@ -0,0 +1,28 @@
+// Note: number of assessments are based on the base data, if that changes, change this test as well
+import { expect } from "chai"
+import MyCounterparts from "../pages/myCounterparts.page"
+import MyTasks from "../pages/myTasks.page"
+
+describe("In my counterparts page", () => {
+  describe("When Erin is checking the content of the page", () => {
+    it("Should see an empty table of pending counterparts", () => {
```

Comment:
How come? In the notifications.spec.js I see:   
```
describe("When checking the notification numbers", () => {
    it("Should see that Erin has 1 counterpart and no tasks that have pending assessments"


```

Reply:
Yep, it should be 1 counterparts, the test case was testing 1, but probably copy paste forgotten :smile: